views/main/results.ejs

```diff
@@ -0,0 +1,10 @@
+<% layout('layouts/boilerplate')%>
+<div class="results-box">
+  <div>
+    <img src="" alt="congratulations" id="results-image" />
```

Comment:
Lose = congratulations
Win = congratulations

🤔

Reply:
We are progressive here and congratulate the player regardless of the result :d
Will fix it though